examples/flower-client/client/flwr_client.py

```diff
@@ -0,0 +1,138 @@
+import argparse
```

Comment:
Add attribution that this comes from the Flower repo, and provide a link back there.